rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/ScheduleAdminController.java

```diff
@@ -58,4 +59,52 @@ public ResponseEntity<?> unscheduleAllServices()throws TException {
         HttpStatus status = HttpStatus.OK;
         return new ResponseEntity<>(requestStatus, status);
     }
+    
+    @RequestMapping(value = SCHEDULE_URL + "/cveService", method = RequestMethod.POST)
+    public ResponseEntity<?> scheduleCve()throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        RequestSummary requestSummary = scheduleService.scheduleCveSearch(sw360User);
+        HttpStatus status = HttpStatus.OK;
+        return new ResponseEntity<>(requestSummary, status);
+    }
+    
+    @RequestMapping(value = SCHEDULE_URL + "/unscheduleCve", method = RequestMethod.DELETE)
+    public ResponseEntity<?> unscheduleCveSearch()throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        RequestStatus requestStatus = scheduleService.cancelCveSearch(sw360User);
+        HttpStatus status = HttpStatus.OK;
+        return new ResponseEntity<>(requestStatus, status);
+    }
+
+    @RequestMapping(value = SCHEDULE_URL + "/deleteAttachment", method = RequestMethod.POST)
+    public ResponseEntity<?> scheduleDeleteAttachment()throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        RequestSummary requestSummary = scheduleService.deleteAttachmentService(sw360User);
+        HttpStatus status = HttpStatus.OK;
+        return new ResponseEntity<>(requestSummary, status);
+    }
+
+    @RequestMapping(value = SCHEDULE_URL + "/unscheduleService", method = RequestMethod.DELETE)
```

Comment:
`/unscheduleService` this sounds generic. Please give a meaningful name.

Reply:
done.